DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/device_global/README.md

```diff
@@ -0,0 +1,224 @@
+
+`device_global` is a class template, parameterized by the type of the underlying allocation, and a list of properties. The type of the allocation also encodes the size of the allocation for potentially multidimensional array types. The list of properties change the functional behavior of the `device_global` instance to enable compiler and runtime optimizations. The `device_image_scope` property is used in the code sample, it limits the scope of a single instance of a `device_global` from a device to a `device_image`, further details on this property can be found [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_device_global.asciidoc#properties-for-device-global-variables). The absence of the `device_image_scope` property is currently not supported by the compiler.
+
+A `device_global` instance can be used to store state across multiple re-launches of a kernel without having to pass in a `buffer` as a kernel argument. An example of an application that would benefit from such a state is where kernels are nodes in a state-machine.
+
+### How to initialize a `device_global` instance
+A `device_global` instance is always zero-initialized. If there is a known first usage of a `device_global` in device code, there exists a technique, described below, that allows a `device_global` to be initialized to a non-zero value. 
+
+Instantiate a second `device_global<bool>` representing a flag that will control when to initialize the `device_global` to a non-zero value. This flag will get zero-initialized to `false`. Once initialization happens, the flag is set to `true` and initialization code doesn't execute on subsequent relaunches of the kernel.
+
+```cpp
+using FPGAProperties = decltype(sycl::ext::oneapi::experimental::properties(
+    sycl::ext::oneapi::experimental::device_image_scope));
+
+sycl::ext::oneapi::experimental::device_global<int, FPGAProperties> val;
+sycl::ext::oneapi::experimental::device_global<bool, FPGAProperties> is_val_initialized;
+
+int main () {
+  sycl::queue q;
+  q.submit([&](sycl::handler& h) {
+    h.single_task([=] {
+      // Initialization happens only once
+      if (!is_val_initialized) {
+        val.get() = 42;
+        is_val_initialized.get() = true;
+      }
+
+      // uses of `val`
+    });
+  });
+}
+```
+
+In a future version of this extension, it is expected that when C++20 support is available and enabled, the `consteval` keyword will be used to enable compile-time constant initialization of the device allocations backing `device_global`. This will simplify some coding patterns, compared with the current zero-initialization requirement.
```

Comment:
We typically avoid future-looking statements in docs. 
Why is this important to tell users here? Do you want them to not use the current non-zero-initialization technique at all and wait for C++20 and `consteval` support? Is there a firm timeline for adding that support?

I suggest removing this line unless the support is planned soon.